src/lib/modules/ens/index.js

```diff
@@ -89,28 +89,41 @@ class ENS {
   }
 
   registerConsoleCommands() {
-    this.embark.registerConsoleCommand((cmd, _options) => {
-      let [cmdName, domain] = cmd.split(' ');
-      return {
-        match: () => cmdName === 'resolve',
-        process: (cb) => global.EmbarkJS.Names.resolve(domain, cb)
-      };
+    this.embark.registerConsoleCommand({
+      use: 'resolve <name>',
+      description: __('Resolves an ENS name'),
+      matches: (cmd) => {
+        let [cmdName] = cmd.split(' ');
+        return cmdName === 'resolve';
+      }
+    }, (cmd, cb) => {
+      let [_cmdName, domain] = cmd.split(' ');
+      global.EmbarkJS.Names.resolve(domain, cb);
```

Comment:
why do you separate the execution from the rest?
if you do: 
```
({
  use: '',
 description: '',
 matches: () => null,
 process: () => null
})
```
I feel the api is more clear and even more readable, also it is only my opinion